src/marketplace/sellers/mftf-vendor.md

```diff
@@ -3,8 +3,12 @@ group: marketplace-sellers
 title: MFTF Vendor-supplied Tests
 ---
 
+{:.bs-callout-info} At this stage, the MFTF Vendor-supplied check is not required to pass technical review. The pass/fail status of this check is ignored. We are still observing the behavior of this set of tests and will start requiring these tests at a later time, with sufficient notice.
```

Comment:
```suggestion
{:.bs-callout-info} At this stage, the MFTF Vendor supplied check is not required to pass technical review. The pass/fail status of this check is ignored. We are still observing the behavior of this set of tests and will start requiring, with sufficient notice, these tests at a later time.
```

Reply:
Magento-supplied and vendor-supplied are two categories of MFTF checks. Omitting the dash makes sentences that use these terms much harder to read. Are we sure we want these changes made?

Reply:
Same. The dashes make sense here.